res/database/Default/user/scr0llbaer.xml

```diff
@@ -0,0 +1,1586 @@
+		
+		<programmerole guid="437539e5-4c6b-4207-981a-051a82021745" first_name="Bobby" last_name="Blaze" gender="1" />
+		<programmerole guid="ba05561a-0266-4cfe-b020-353781fd881c" first_name="Xorp" last_name="" gender="0" />
+	</programmeroles>
+
+	<scripttemplates>
+		<scripttemplate product="3" licence_type="1" guid="e569b0af-e97f-4897-8b8d-6c426fa5a7d5" creator="9551" comment="Low-Budget- und Endlos-Variante von scripttemplate-ron-morningshow">
+			<title>
+				<de>%title%</de>
+				<en>%title%</en>
+			</title>
+			<description>
+				<de>Programmfüller für den jeden Morgen. Hier werden nochmal die Beiträge aus dem Mülleimer gefischt, die die Kollegen aus der Nachrichtenredaktion schon weggeschmissen haben. Andere niedrigschwellige Inhalte finden sich bestimmt auch noch.</de>
+				<en>Programme filler for every morning. Here the stories that the colleagues from the newsroom have already thrown away are fished out of the garbage can again. Other low-threshold content can surely be sourced as well.</en>
+			</description>
+
+			<variables>
+				<title>
+					<de>Morgenmuffel-TV|Mundgeruch am Morgen|Miesepeter am Morgen|Morgengrieß-Kram|G'n Morgääähn</de>
+					<en>Morning Grouch TV|Bad breath in the Morning|Grumpy in the Morning|Morning Flour and Stuff|G'd Moryaaawn</en>
+				</title>
+			</variables>
+
+			<jobs>
+				<job index="0" function="8" required="1" />
+			</jobs>
+			<data flags="1" broadcast_time_slot_start="5" broadcast_time_slot_end="10" production_limit="100" production_broadcast_limit="1" production_broadcast_flags="512" production_licence_flags="36" />
+			<genres maingenre="103" subgenres="6,300" />
+			<blocks value="3" />
+			<price min="2000" max="4000" />
+			<potential min="20" max="40" />
+			<outcome min="20" max="40" />
+			<review min="10" max="20" />
+			<speed min="10" max="20" />
+			<production_time value="60" />
+		</scripttemplate>
+
+		<scripttemplate product="3" licence_type="1" guid="d35ad24a-9095-490e-b5d1-f51ba82210ff" creator="9551" comment="Deluxe- und Endlos-Variante von scripttemplate-ron-morningshow">
+			<title>
+				<de>%title%</de>
+				<en>%title%</en>
+			</title>
+			<description>
+				<de>Es lohnt sich! Für jeden Morgen! Tolles Programm und gute Unterhaltung für einen beschwingten Start in den Tag.</de>
+				<en>It's worth it! For every morning! Great program and good entertainment for an upbeat start into the day.</en>
+			</description>
+
+			<variables>
+				<title>
+					<de>Das saftige Frühstücks-TV|5-Sterne Frühstück Deluxe|Morgenpracht-TV|Croissant mit Latte am Morgen TV</de>
+					<en>The Lush Breakfast TV|5-Star Breakfast Deluxe|Morning Glory TV|Croissant with Latte in the Morning TV</en>
+				</title>
+			</variables>
+
+			<jobs>
+				<job index="0" function="1" required="1" />
+				<job index="1" function="4" required="1" />
+				<job index="2" function="8" required="1" />
+				<job index="3" function="128" required="1" />
+				<job index="4" function="2048" gender="2" required="1" />
```

Comment:
Have you tried if a model is actually castable? At least currently it is not part of the selectable jobs.

Reply:
It's generally working but the job name is not properly displayed:

![Screenshot from 2023-02-27 04-20-17](https://user-images.githubusercontent.com/123826268/221474360-a0574a9b-b2bf-4df8-8357-51a6d8448571.png)

But it can be set (no people exist besides amateurs though).

However, I'm commenting it out for now, because `required="0"` isn't working. I'm obliged to fill the job even though it's supposed to be optional.

Reply:
The translation is missing because it is no "Cast Job". This *may* also be the reason that the optional flag is not working.
You can verify this by setting all jobs to optional.

Reply:
Should we add support for the "non cast" stuff? If so I guess we should do via a new "issue"

Reply:
I would not allow non-cast jobs at the moment. It is not just about translating the job titles. To do it properly, we would have to add persons for the corresponding jobs and incorporate the "matching"-logic (what if you cast an old ugly man for a female model.)

There are enough open issues with respect to custom productions, already. And the question of "what about the other 20 jobs that would be nice to have" was already raised.

Reply:
This is why I asked for raising a new issue (or using an existing open one). 

The idea of having "other people" in a production is of course intruiging (but leads to the "I need more jobs" issue) as it is needed for the well known "show" formats with VIP guests

Reply:
This issue can be tracked via #885